llvm/test/CodeGen/DirectX/ShaderFlags/double-extensions.ll

```diff
@@ -12,6 +12,13 @@ target triple = "dxil-pc-shadermodel6.7-library"
 ; CHECK-NEXT: ;
 ; CHECK-NEXT: ; Shader Flags for Module Functions
 
+;CHECK: ; Function top_level : 0x00000044
+define void @top_level() #0 {
+  call void @test_uitofp_i64(i64 noundef 5)
```

Comment:
There is no `void @test_uitofp_i64`, and it isn't clear to me what noundef is doing here. I think you meant `call double @test_uitofp_i64(i64 5)`

Reply:
> There is no `void @test_uitofp_i64`, and it isn't clear to me what noundef is doing here. I think you meant `call double @test_uitofp_i64(i64 5)`

Changed.